### ISSUE TITLE
Car Extras + Gas

### ISSUE THREAD

**Issue by ZeroInsomniac** [New-Chance-RP/Feedback#448]
<h2>When you have extras toggled off, then you get gas, your car fixes itself and puts said extras back onto the vehicle</h2>

<h2>To just fill up the tank</h2>

<h2>I'll grab a video if I can reproduce it next time I get gas</h2>

**Comment by Senlarus** [comment]
Yeah, this has to do with how the extras work in GTA. By default, every time you change an extra on a vehicle it resets the model of the vehicle including ALL properties. This includes damage and gas. So the script I have disables the auto-repairing upon changing extras which will result in the options that you have selected not saving until you repair the vehicle. Once repaired, it should keep those extras on no matter what until you store and retrieve the vehicle again. I do plan on fixing this with an update to the extras menu in a future update, but I needed to get on to other things so I pushed it the way it is for now.

I will leave this ticket, and update it, as a to-do to enhance the current vehicle extras script.
